index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Challenge</title>
+  <link rel="stylesheet" href="styles.css" </head>
+  <script src="challenge.js"></script>
+
+<body>
+  <h1>Adam Smith</h1>
+  <p id="bio">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos <a
+      href="https://www.linkedin.com">linkedin </a>ipsum, ducimus saepe ratione repudiandae, rem ad doloribus quas
+    quidem alias dicta doloremque mollitia! Explicabo vel, earum vero velit nobis cupiditate.</p>
+  <h2>Education history</h2>
```

Comment:
I request you to add this block of code in a div so that you have more control on the position of your project 😄